README.md

```diff
@@ -1,3 +1,149 @@
-# React Native Base
+# React Native Base - TypeScript
 
-This new base is under construction
+## Commands
+
+1. **Start the app**: `yarn android:{env}` or `yarn ios:{env}` (envs: `dev`, `staging`, and `prod`)
+2. **Start metro bundler**: `yarn start`
+3. **Lint the app**: `yarn lint`
+4. **Test the app**: `yarn test`
+
+## Getting started
+
+1. Install dependencies: `yarn`
+2. Create a `.env` file in the root directory of the project, based on the `.env.defaults` sample file and the extra constants that you may need. This will be your development env.
+
+You can also create `.env.prod` and `.env.staging` to define environment variables for production and staging.
+
+3. Rename your new project using `yarn rename`
+4. Start on android or ios: `yarn android:{env}` or `yarn ios:{env}` (envs: `dev`, `staging`, and `prod`)
+
+### Steps for Android development
+
+None
+
+### Steps for iOS development
+
+1. Run the following command inside the `ios` directory
+
+```
+pod install
```

Comment:
Should we start using `npx pod-install` instead?

https://www.npmjs.com/package/pod-install